.github/workflows/e2e.yaml

```diff
@@ -20,12 +20,13 @@ on:
     types:
       - completed
     branches:
+      - "pull-request/[0-9]+"
       - main
       - release-*
 
 jobs:
   e2e-tests:
-    runs-on: ubuntu-latest
+    runs-on: linux-amd64-cpu4
     if: ${{ github.event.workflow_run.conclusion == 'success' }} && ${{ github.event.workflow_run.event == 'push' }}
```

Comment:
@elezar We could remove this now if we want. and just run the test during the PR, we can control when to run the E2E test by dropping a comment `ok-to-test`

Reply:
If we don't remove this, then we're not going to see this status on PRs, correct?

Reply:
correct

Reply:
It's up to you to do this as a follow up or as part of this PR.